source/frontend/src/features/mapSideBar/property/tabs/propertyDetails/detail/propertyOperation/OperationContainer.tsx

```diff
@@ -107,6 +109,7 @@ export const OperationContainer: React.FunctionComponent<IOperationContainerProp
     <View
       subdivisionOperations={subdivisionOperations}
       consolidationOperations={consolidationOperations}
+      loading={!propertyId || getPropertyOperations.loading}
```

Comment:
nit: I think we are moving towards a bit more strict checking for nulls/undefined - ie `!exists(propertyId)`